rdkit-stubs/Chem/rdDistGeom.pyi

```diff
@@ -38,7 +41,26 @@ def ETDG() -> Any: ...
 def ETKDG() -> Any: ...
 def ETKDGv2() -> Any: ...
 def ETKDGv3() -> Any: ...
-def EmbedMolecule(RDKit) -> Any: ...
+def EmbedMolecule(
+    mol: Mol,
+    maxAttempts: int = 0,
+    randomSeed: int = -1,
+    clearConfs: bool = True,
+    useRandomCoords: bool = False,
+    boxSizeMult: float = 2.0,
+    randNegEig: bool = True,
+    numZeroFail: int = 1,
+    coordMap: dict[int, Point3D] = {},
```

Comment:
isn't this a mutable default? Perhaps that is safe in `.pyi` files?

Reply:
Not sure what you mean by mutable default.

Previous discussion: https://github.com/postera-ai/rdkit-stubs/pull/1#discussion_r1198374421

Reply:
Note `EmbedMolecule` is a function, not a class.

Reply:
https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

Reply:
Ah, I see. Nice to know about that footgun. Should be ok here since stubs have no effect at runtime. Also I don't see a good alternative.

Sidenote, setting the default to `None` is probably not the right workaround in 2023 since it changes the type of the argument. Also, default arguments serve as documentation and setting them to `None` removes that info.